crates/project/src/terminals.rs

```diff
@@ -174,6 +174,8 @@ impl Project {
                     command_label: spawn_task.command_label,
                     hide: spawn_task.hide,
                     status: TaskStatus::Running,
+                    hide_task: spawn_task.hide_task,
```

Comment:
I don't mean to bikeshed, but I think it's potentially confusing for `hide` to control the task terminal tab behavior and `hide_*` to suppress certain output inside the terminal. E.g:

```
  {
    "label": "Echo Echo",
    "command": "echo ECHO; sleep 1; echo echo; sleep 1",
    "hide": "never",
    "hide_task": true,
    "hide_command": true
  },
```

Looks like vscode has an entire `presentation` key (your `hide_command` corresponds to `presentation.echo = false`):
```json
//defaults
"presentation": {
    "echo": true,
    "reveal": "always",
    "focus": false,
    "panel": "shared",
    "showReuseMessage": true,
    "clear": false
},
```

Perhaps substitue:
- `show_command` for `hide_command` (default true)
- `show_summary` for `hide_task` (default true)

We should also document these: [tasks.md](https://github.com/zed-industries/zed/blob/main/docs/src/tasks.md?plain=1)

Reply:
Otherwise this is awesome.
I didn't realize I wanted clean task output until I tried it. 👍

Reply:
Hello, thank you for the feedback.
After seeing your comment I realize how much sense it makes.

Also, learned a new expression - bikeshed (not native English speaker)